index.bs

```diff
@@ -17,6 +17,8 @@ spec:dom; type:interface; for:/; text:Document
 spec:url; type:dfn; for:url; text:origin
 spec:fetch; type:dfn; for:Response; text:response
 spec:html; type:dfn; for:/; text:browsing context
+spec:html; type:dfn; for:/; text:navigable container
+spec:html; type:dfn; for:/; text:nested navigable
```

Comment:
These shouldn't be necessary, but I guess HTML failed to export these terms. Will fix!

You'll need to change nested navigable references to `[=navigable container/nested navigable=]`, when I do.

Reply:
Done!